### DISCUSSION TITLE
Switching from one time payment to subscription

### DISCUSSION THREAD

**Discussion by torerik123** [Glench/ExtPay#239] [General]
I want to switch from lifetime to a monthly subscription.  
What is the best way to do this? 
Do I have to create a new extpay project?
The users that have already paid for lifetime should keep their plan, but all new users should be signed up for a subscription plan.

**Comment by Glench** [comment]
You can now do this very easily on the ExtensionPay website! Or have multiple plans if you like with the newest version of ExtPay.
